bundles/http_admin/CMakeLists.txt

```diff
@@ -17,9 +17,9 @@
 
 celix_subproject(HTTP_ADMIN "Service to use a HTTP server with websocket support" ON)
 if (HTTP_ADMIN)
+    find_package(civetweb REQUIRED)
 
     add_subdirectory(http_admin_api)
-    add_subdirectory(civetweb)
```

Comment:
Note this removes the cmake target `Celix::civetweb_shared` and that is a backwards incompatible update.

I think reintroducing a dummy shared lib `Celix::civetweb_shared`, for example with a single dummy c function `void celix_civetweb_dummy()` is needed for Celix 2.x.
This was downstream users still can include the `Celix::civetweb_shared` as private bundle lib, but actually depend on the `civetweb::civetweb` through the interface lib `Celix::http_admin_api`

This can be removed when we move towards a Celix 3.0.0 release.